spec/index.bs

```diff
@@ -1918,6 +1921,7 @@ enum GPUCompareFunction {
         - |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}} is greater than or equal to 0.
         - |descriptor|.{{GPUSamplerDescriptor/lodMaxClamp}} is greater than or equal to
             |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}}.
+        - |descriptor|.{{GPUSamplerDescriptor/maxAnisotropy}} is between 1 and 16, inclusive.
```

Comment:
I don't think this needs a specified max, even though it's not larger than 16x in practice.